### ISSUE TITLE
Explaining `fetch` command via the word "pull" may confuses the newbie

### ISSUE THREAD

**Issue by YueLinHo** [progit/progit2#530]
book/**02**-git-basics/sections/**remotes.asc** line 114

Since `pull` is a command in git and is different with `fetch`, it may confuses the newbie while indroducing the pull command later??? or not?

```
It's important to note that the `git fetch` command pulls the data to your local repository [...]

[...] you can use the `git pull` command to automatically fetch and then merge a remote branch into your current branch.(((git commands, pull)))
```

Would it better to use `download` or `just fetch` for `pulls`?

**Comment by ben** [comment]
Ooh, nice. You're on a roll! Want to submit another PR?

**Comment by YueLinHo** [comment]
Do you prefer **one PR** for my 3 recent issues which are belong to the same section?

If so, perhaps re-use the first PR #528, then edit the title. Sure thing, 3 commits there. 

Otherwise, I am going to submit separated PR for each issue.

**Comment by YueLinHo** [comment]
Fixed in PR #538.
